content/what-is/top-iac-tools.md

```diff
@@ -0,0 +1,129 @@
+- **Ecosystem**: Building its community and ecosystem, leveraging its Terraform roots.
+
+## Cloud-Specific IaC Tools
+
+While multi-cloud tools offer flexibility and prevent vendor lock-in, many organizations still opt for cloud-specific IaC tools.
+
+Cloud-specific tools can offer:
+
+1. Potentially tighter integration with native services
+2. Potentially simpler setup for teams already familiar with a particular cloud ecosystem
+3. Access to platform-specific features that might not be available in multi-cloud tools
+
+However, it's important to note that multi-cloud solutions offer more strategic advantages. They prevent vendor lock-in, maintain flexibility, and promote skill portability across different cloud environments. Despite these benefits of multi-cloud tools, understanding cloud-specific options is crucial for a comprehensive view of the IaC tooling landscape.
+
+Let's explore some of the most prominent cloud-specific IaC tools and how they compare to their multi-cloud counterparts in terms of flexibility, integration, and ecosystem support.
+
+### AWS CloudFormation
+
+AWS CloudFormation supports JSON or YAML. Like Pulumi and Terraform, it has a declarative, desired state approach.
+
+CloudFormation is supported by extensive documentation. However, managing complex templates can be challenging, and there is a steep learning curve associated with CloudFormation's syntax and AWS services. Performance issues may arise with large deployments, but rollback triggers can revert to a previous state if a deployment fails.
+
+CloudFormation supports cross-account and cross-region deployments. Despite these strengths, CloudFormation has limitations. Template size limitations and cryptic error messages complicating error handling are among the most common online complaints. Managing dependencies between resources can become complex past a certain scale, and testing capabilities for templates are limited.
+
+- **Flexibility**: Some support for abstraction and modularity through nested stacks and reusable templates.
+- **Integration**: Integrates well with AWS-native CI/CD. Broader integration to non-AWS environments is possible but more challenging.
+- **Ecosystem**: Supported by documentation, providing resources for learning and troubleshooting.
+
+### Azure Resource Manager
+
+Azure Resource Manager (ARM) supports JSON-based configuration or using Bicep - a ARM-specific DSL. JSON-based templating is also possible. ARM integrates deeply with Azure services including Azure DevOps, however, integration with non-Azure workflows can be more challenging. The ecosystem is supported by a large community and extensive documentation. The complexity of managing dependencies between resources with ARM can increase with scale. Potential performance issues with large deployments have been a challenge with ARM. However, Azure Resource Manager is preferred by those who want a native Azure tool and are deeply invested in the Azure ecosystem.
```

Comment:
```suggestion
Azure Resource Manager (ARM) supports JSON-based configuration or using Bicep - a ARM-specific DSL. JSON-based templating is also possible. ARM integrates deeply with Azure services including Azure DevOps. However, integration with non-Azure workflows can be more challenging. The ecosystem is supported by a large community and extensive documentation. The complexity of managing dependencies between resources with ARM can increase with scale. Potential performance issues with large deployments have been a challenge with ARM. However, Azure Resource Manager is preferred by those who want a native Azure tool and are deeply invested in the Azure ecosystem.
```

"JSON-based templating is also possible." Is this a redundant sentence?

" integration with non-Azure workflows can be more challenging." Can you provide more details on how?

"The complexity of managing dependencies between resources with ARM can increase with scale." Can you explain how this is the can in comparison to Pulumi?

"Potential performance issues with large deployments have been a challenge with ARM." I think this could benefit from more details as well.